content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -497,22 +497,24 @@ datasource db {
 model User {
   email   String    @unique
   name    String?
-  user_id Int       @default(autoincrement()) @id
+  user_id Int       @id @default(autoincrement())
   post    Post[]
   profile Profile[]
 }
 
 model Post {
   content   String?
-  post_id   Int     @default(autoincrement()) @id
+  post_id   Int     @id @default(autoincrement())
   title     String
-  author_id User?
+  author    User?   @relation(fields: [author_id], references: [user_id])
+  author_id Int?
```

Comment:
Whoah, how does our formatter do this to fix this broken schema? Or was this a manual thing @umidbekk?

Reply:
@janpio It fixes schema automatically, but requires some manual work afterwards

![demo](https://raw.githubusercontent.com/umidbekk/prettier-plugin-prisma/main/demo.gif)